panel/src/components/Forms/Input/ListInput.vue

```diff
@@ -35,18 +43,22 @@ export default {
       let list = dom.querySelector('ul, ol');
 
       if (!list) {
-        this.$emit("input", "");
+        this.$emit("input", this.list = "");
```

Comment:
We might be able to skip this change and the one below, when we add a `watch` on `value` and update `this.list` whenever the `value` prop changes. We might need to do this anyways to avoid unforeseen states.

Reply:
If we bind the `value` in watch to `list`, it will not work, due to following line:

https://github.com/getkirby/kirby/blob/7a5707ff2426b843ff40084e4d2c99a94bda418c/panel/src/components/Forms/Input/ListInput.vue#L60-L61

For better understanding:

`list` prop: raw html that includes `p` tags
`value` prop: cleared html that doesn't includes `p` tag